<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_produk</class>
 <widget class="QWidget" name="Form_produk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #d8d8d8;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_header">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border-radius: 15px;
border: 1px solid #ddd;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_logo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Jersey Sharp</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>UNIFEST</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dashboard">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Milky Nice</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff5757;
color: white;</string>
     </property>
     <property name="text">
      <string>DASBOARD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_produk">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Milky Nice</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff5757;
color: white;</string>
     </property>
     <property name="text">
      <string>PRODUK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_supplier">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Milky Nice</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff5757;
color: white;</string>
     </property>
     <property name="text">
      <string>SUPPLIER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_customer">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Milky Nice</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff5757;
color: white;</string>
     </property>
     <property name="text">
      <string>CUSTOMER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_penjualan">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Milky Nice</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff5757;
color: white;</string>
     </property>
     <property name="text">
      <string>PENJUALAN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_menu">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Milky Nice</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff5757;
color: white;</string>
     </property>
     <property name="text">
      <string>logout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pembelian">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Milky Nice</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff5757;
color: white;
</string>
     </property>
     <property name="text">
      <string>PEMBELIAN</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>851</width>
      <height>641</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>10000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>821</width>
       <height>291</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>240</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ff5757;
color: white;
padding: 10px;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">Font size: 20â€“24 pt;</string>
      </property>
      <property name="text">
       <string>Kelola Produk</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>791</width>
        <height>206</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_nama_produk&#10;_2">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nama Produk</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="input_nama_produk">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ff6f6f;
color: white;
padding: 6px 12px;
border-radius: 8px;

</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Lbl_harga_beli_2">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Harga Beli</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="input_harga_beli">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ff6f6f;
color: white;
padding: 6px 12px;
border-radius: 8px;

</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Lbl_Harga_Jual_2">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Harga Jual</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="input_harga_jual">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ff6f6f;
color: white;
padding: 6px 12px;
border-radius: 8px;

</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Lbl_Stok_2">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stok</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="input_stok">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ff6f6f;
color: white;
padding: 6px 12px;
border-radius: 8px;

</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="supplier_Label_2">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Supplier</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="combo_supplier">
         <property name="font">
          <font>
           <family>Milky Nice</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ff6f6f;
color: white;
padding: 6px 12px;
border-radius: 8px;

</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_Simpan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
color: black;</string>
      </property>
      <property name="text">
       <string>Simpan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>250</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
color: black;</string>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Hapus">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
color: black;</string>
      </property>
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Reset">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
color: black;</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>821</width>
       <height>301</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>350</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User</string>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nama Produk</string>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Harga Beli</string>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Harga Jual</string>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stok</string>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Supplier</string>
      </property>
      <property name="font">
       <font>
        <family>Milky Nice</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
